js/src/main/scala/scala/json/ast/JValue.scala

```diff
@@ -31,6 +31,34 @@ sealed abstract class JValue extends Product with Serializable {
   def toJsAny: js.Any
 }
 
+@JSExport
+object JValue {
+  /**
+    * Converts a Javascript object/value coming from Javascript to a [[JValue]].
```

Comment:
I revered the doc for [toJsAny](https://github.com/mdedetrich/scala-json-ast/pull/7/files#diff-bbea03361f116097645be6b8f3000a28R31), but I'm not sure I understand the nuance here, isn't it the case that `js.Any` correspond to both Javascript values and Scala.js Javascript values?